### ISSUE TITLE
I'm ready to dump Alexa

### ISSUE THREAD

**Issue by smartypantsuk** [bwssytems/ha-bridge#1335]
They really are the shittiest devices developed by muppets. In the last 5 years they before worse not better. But ranting is not why I'm here now.

I have strange issues that never used to be a problem.  I've noticed that when I'm controlling fibaro lights via Alexa (through ha-bridge) then everything works fine with respect to alexa acknowledgement.

But when I'm switching on virtual devices (in homeseer) via the same method,  then Alexa will switch the device on,  but complain that the device isn't responding.  If I turn off virtual devices then Alexa responds properly. 

To summarise.  Turning on and off physical devices through Alexa it works properly and shreds resonds properly. Turning OFF virtual devices Alexa resonds properly too.  But Turning ON virtual device using the same commands in ha-bridge, then alexa complains that the device isn't responding but turns it on anyway. 

I can only assume Alexa isn't receiving a response in a acceptable time or not one at all.

This all used to work fine.  Everything on the same subnet and firewall off for testing.

**Comment by smartypantsuk** [comment]
> I know I'm late to the party, but in case somebody stumbles over the same issue:
> 
> This is exactly what happens if devices are discovered twice by Alexa, once as a "Dimmable Light" and once as a "Royal Philips Electronics intelligent device". it's the Philips devices which express this behavior. Just get rid of the Philips devices and only leave the dimmable lights. All will be good then.
> 
> I don't know why those Philips devices turn up occasionally and why they don't always cause issues, but if the issues happen it's always been that.
> 
> Alex

But, all my devices are virtual devices,  all appear in alexa as Royal Philips Electronics intelligent device as usually work ok. I do think the way alexa works with multiple alexas in the environment is rubbish,  there is definitely something badly designed here. 

I'd use Google devices but currently havn't a way to do what I want easy
